deploy/backplane/mcs-tier-two/hypershift/management-cluster/20-mcs-tier-two.SubjectPermission.yml

```
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mcs-tier-two-management-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-mcs-tier-two-management-cluster-cluster
+  - view
```

```suggestion
  - backplane-readers-cluster
```

Inconsistent with docs.